src/common/AdaptiveTree.h

```diff
@@ -505,7 +507,10 @@ class ATTR_DLL_LOCAL AdaptiveTree
   bool adp_timelined_, period_timelined_;
 
   bool current_hasRepURN_, current_hasAdpURN_;
-  std::string current_pssh_, current_defaultKID_, current_iv_;
+  std::string current_pssh_;
+  std::string current_defaultKID_;
+  std::string current_iv_;
+  CryptoMode m_currentCryptoMode;
```

Comment:
i know that it was existing code but i have not understand because there is a prefix with "current" when there is a fixed value, i am wrong?
in case i would prefer to drop this prefix